### PR TITLE
Add pylint as a mandatory unit test

### PR DESCRIPTION
**Pull request by 2Cubed**
Fix existing pylint errors. Satisfies #171.

### CODE REVIEW

**Comment by 2Cubed** [comment]
Note: `pylint` will not yet past, due to `# TODO` about `BanPacket`.

File: cactusbot/api.py [hunk 1]
```diff
@@ -26,7 +26,20 @@ def __init__(self, token, password, url=URL, auth_token="", **kwargs):
         self.token = token
         self.auth_token = auth_token
         self.password = password
-        self.URL = url
+        self.url = url
+
+        self.buckets = {
```
Comment (Innectic) [inline cactusbot/api.py:31]: This is a really neat system :o

File: cactusbot/commands/command.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# pylint: skip-file
```
Comment (Innectic) [inline cactusbot/commands/command.py]: So much cheating

**Review by Innectic — APPROVED** [review]

File: cactusbot/api.py (hunk 1 above)
Comment (2Cubed) [reply cactusbot/api.py:31]: Thanks! Much cleaner than what we had before - it's nice to have things grouped together, too. :smile:

File: cactusbot/commands/command.py (hunk 2 above)
Comment (2Cubed) [reply cactusbot/commands/command.py]: Yeah... looking back on it, that's a bit too much cheating. Switching to individual errors. :+1:

**Pull request merged**